scvi/model/_multivi.py

```diff
@@ -0,0 +1,709 @@
+        of `n_hidden`.
+    n_layers_encoder
+        Number of hidden layers used for encoder NNs.
+    n_layers_decoder
+        Number of hidden layers used for decoder NNs.
+    dropout_rate
+        Dropout rate for neural networks
+    model_depth
+        Model sequencing depth / library size (default: True)
+    region_factors
+        Include region-specific factors in the model (default: True)
+    latent_distribution
+        One of
+        * ``'normal'`` - Normal distribution (Default)
+        * ``'ln'`` - Logistic normal distribution (Normal(0, I) transformed by softmax)
+    deeply_inject_covariates
+        Whether to deeply inject covariates into all layers of the decoder. If False (default),
+        covairates will only be included in the input layer.
+    fully_paired
+        allows the simplification of the model if the data is fully paired. Currently ignored.
+    **model_kwargs
+        Keyword args for :class:`~scvi.module.PEAKVAE`
+
+    Notes
+    -----
+    * the model assumes that the features are organized so that all expression features are
+    consequtive, followed by all accessibility features. For example, if the data has 100 genes
+    and 250 genomic regions, the model assumes that the first 100 features are genes, and the
+    next 250 are the regions.
+
+    * the main batch annotation, specified in the `scvi.data.setup_anndata`, should correspond to
+    the modality each cell originated from. This allows the model to focus mixing efforts, using
+    an adversarial component, on mixing the modalities. Other covariates can be specified using
+    the `categorical_covariate_keys` argument.
+
+    Examples
+    --------
+    >>> adata_rna = anndata.read_h5ad(path_to_rna_anndata)
+    >>> adata_atac = scvi.data.read_10x_atac(path_to_atac_anndata)
+    >>> adata_multi = scvi.data.read_10x_multiome(path_to_multiomic_anndata)
+    >>> adata_mvi = scvi.data.organize_multiome_anndatas(adata_multi, adata_rna, adata_atac)
+    >>> scvi.data.setup_anndata(adata_mvi, batch_key="modality")
+    >>> vae = scvi.model.MULTIVI(adata_mvi)
+    >>> vae.train()
+    """
+
+    def __init__(
+        self,
+        adata: AnnData,
+        n_genes: int,
+        n_regions: int,
+        n_hidden: Optional[int] = None,
+        n_latent: Optional[int] = None,
+        n_layers_encoder: int = 2,
+        n_layers_decoder: int = 2,
+        dropout_rate: float = 0.1,
+        region_factors: bool = True,
+        gene_likelihood: Literal["zinb", "nb", "poisson"] = "zinb",
+        use_batch_norm: Literal["encoder", "decoder", "none", "both"] = "none",
+        use_layer_norm: Literal["encoder", "decoder", "none", "both"] = "both",
+        latent_distribution: Literal["normal", "ln"] = "normal",
+        deeply_inject_covariates: bool = False,
+        encode_covariates: bool = False,
+        fully_paired: bool = False,
+        **model_kwargs,
+    ):
+        super(MULTIVI, self).__init__(adata)
```

Comment:
```suggestion
        super().__init__(adata)
```

just for consistency